modules/administration-guide/pages/installing-che-using-storage-classes.adoc

```diff
@@ -46,43 +36,17 @@ Use CheCluster Custom Resource definition to define storage classes:
 spec:
   devEnvironments:
     storage:
-      pvc:
-        # keep blank unless you need to use a non default storage class for workspace PVC(s)
-        storageClass: 'workspace-storage'
-----
-
-. Define the persistent volume for a {prod-short} workspace in a `che-workspace-pv.yaml` file:
-+
-.`che-workspace-pv.yaml` file
-[source,yaml]
-----
-apiVersion: v1
-kind: PersistentVolume
-metadata:
-  name: workspace-pv-volume
-  labels:
-    type: local
-spec:
-  storageClassName: workspace-storage
-  capacity:
-    storage: 10Gi
-  accessModes:
-    - ReadWriteOnce
-  hostPath:
-    path: "/data/che/workspace"
-----
-
-. Bind the persistent volume:
-+
-[subs="+quotes,+attributes"]
+      perUserStrategyPvcConfig:
```

Comment:
@tolusha can we add more yaml here? `spec:storage:` I mean to make it clear what part of CR is expected to be updated

Reply:
Diff confuses, but the whole CR is there:

![screenshot-64b12ece30412d0336828eb1--eclipse-che-docs-pr netlify app-2023 07 17-14_53_19](https://github.com/eclipse-che/che-docs/assets/1640675/54922c68-e251-4ad0-9eb7-7d8ea9ae1280)